lib/frameworks/debugprint.ts

```diff
@@ -1,7 +1,8 @@
-var util = require('util'),
-    Logger = require('../logger').Logger;
+import * as util from 'util';
+import {Logger} from '../logger';
+import {Runner} from '../runner';
 
```

Comment:
`import {RunResults} from '../taskRunner';

We might need to update the RunResults to be more flexible:
https://github.com/angular/protractor/blob/selenium4/lib/taskRunner.ts#L9

```
export interface RunResults {
  taskId?: number;
  specs?: Array<string>;
  ...
```